aggregator_api/src/models.rs

```diff
@@ -23,6 +23,7 @@ pub(crate) enum AggregatorRole {
 
 #[derive(Serialize, PartialEq, Eq, Debug)]
 pub(crate) struct AggregatorApiConfig {
+    pub dap_versions: &'static [&'static str],
```

Comment:
I think we should use a schema where the capabilities can vary by DAP version. i.e. something like `{"DAP-04": {/*remainder of capabilities*/}, "DAP-05": {/*remainder of capabilities*/}}`. This is because DAP capabilities are likely to vary by version, both in terms of what capabilities need to be represented by a given version, and by which capabilities are supported by any given implementation on a per-version basis.

Reply:
I continue to disagree on this.

Reply:
Marking this as resolved, as the current PR only attempts to address single-protocol-version deployments